go.mod

```diff
@@ -6,3 +6,7 @@ require (
 	github.com/grailbio/go-dicom v0.0.0-20190117035129-c30d9eaca591
 	github.com/stretchr/testify v1.2.2
 )
+
+replace (
+	github.com/grailbio/go-dicom => ../go-dicom
+)
```

Comment:
this should be reverted.

Reply:
Updated at [0e38eeb](https://github.com/segmed/go-netdicom/pull/14/commits/0e38eeb0002764678375820267c3474e804efbb2)